src/tests/Main.ts

```diff
@@ -440,11 +440,21 @@ describe("Scry", function () {
 		});
 	});
 
-	it("homepage links", async () => {
-		const result = await Scry.Misc.homepageLinks();
-		expect(result).satisfies(Array.isArray);
-		expect(Scry.error()).eq(undefined);
-	});
+  describe("Misc", () => {
+    it("bulk data", async () => {
+      const result = await Scry.Misc.bulkData();
+
+      expect(result).satisfies(Array.isArray);
+      expect(result.length).eq(5);
+      expect(Scry.error()).eq(undefined);
+    });
+
+    it("homepage links", async () => {
+      const result = await Scry.Misc.homepageLinks();
+      expect(result).satisfies(Array.isArray);
+      expect(Scry.error()).eq(undefined);
+    });
+  })
```

Comment:
Fix indent, add missing semicolon, replace bulk data length check with `gte` for a bit of future proofing
```suggestion
	describe("Misc", () => {
		it("bulk data", async () => {
			const result = await Scry.Misc.bulkData();

			expect(result).satisfies(Array.isArray);
			expect(result.length).gte(5);
			expect(Scry.error()).eq(undefined);
		});

		it("homepage links", async () => {
			const result = await Scry.Misc.homepageLinks();
			expect(result).satisfies(Array.isArray);
			expect(Scry.error()).eq(undefined);
		});
	});
```

Reply:
Good point.